### DISCUSSION TITLE
Light house integration with Cypress latest version

### DISCUSSION THREAD

**Discussion by Vamsi-K-Reddy** [cypress-io/cypress#23686] [Plugins]
Hi,

Any one tried out integrating light house(for performance tests) with latest versions of Cypress(above 10).
Please detail the steps followed if you have successfully integrated.
(https://www.npmjs.com/package/cypress-lighthouse)

Thanks.

**Comment by emilyrohrbough** [comment]
@Vamsi-K-Reddy The Cypress team typically doesn't actively participate in discussion threads unless they are tied to new features or experiments. You'll likely have better luck getting feedback/stimulated a discussion on this topic out in discord: https://on.cypress.io/chat.
